contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -92,7 +82,6 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
     uint256 internal constant MAX_UINT128 = 2**128 - 1;
```

Comment:
Minor: Can get rid of `struct PositionList` declaration (above).

Reply:
done